lib/her/api.rb

```diff
@@ -89,19 +89,26 @@ def request(opts={})
       path = opts.delete(:_path)
       headers = opts.delete(:_headers)
       opts.delete_if { |key, value| key.to_s =~ /^_/ } # Remove all internal parameters
-      response = @connection.send method do |request|
+      # Faraday doesn't support the OPTIONS verb because of a name collision with an internal options method
```

Comment:
do we want to put these comments underneath `if method == :options` like we do for `if method == :get`?